lms/static/sass/shared/_alerts_pattern_library_shim.scss

```diff
@@ -60,31 +71,62 @@ $alert-border: 0.0625rem solid $alert-border-grey;
 
 // everything below here gets added specificity pattern-library-shim
 .pattern-library-shim {
-
     // ----------------------------
     // #GENERAL
     // ----------------------------
-
     &.alert {
-        background-color: $alert-background-color;
-        border: $alert-border;
-        border-radius: $alert-border-radius;
-        border-top-left-radius: 0;
-        border-top-right-radius: 0;
-        padding: 1.25rem;
+        background-color: $white;
```

Comment:
[The UXPL is using $white here](https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_alerts.scss#L44), isn't it better to keep differences between that file and this one to a minimum?